### PR TITLE
Feature/tweet

### PR DESCRIPTION
**Pull request by ZimGil**
## Feature: Tweet

* Getting twitter auth tokens from `process.env`
* Verifying account
* Posting `tweet` at a Twitter update (tweet)

All errors are logged to the console for now.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+  access_token_key: process.env.ACCESS_TOKEN_KEY,
+  access_token_secret: process.env.ACCESS_TOKEN_SECRET
+});
+
+client
+  .get("account/verify_credentials")
+  .then(() => {
+    console.log('account verified');
+    return client.post('statuses/update', {status: tweet});
+  }).then(() => console.log('Tweeted:', tweet))
```
Comment (Thatkookooguy) [inline src/index.ts]: very small nit, but if the previous `.then(` and the next `.catch(` are starting on a newline, I suggest this `.then(` should start on a new line as well

Also, we should probably integrate a linter in this project. do you want to add tslint? I can do it for you if you prefer

Comment (Thatkookooguy) [reply src/index.ts]: (I said the comment about the linter since I saw you use both single quotes and double quotes :-))

Comment (ZimGil) [reply src/index.ts]: The double quotes happened due to a copy and paste.

I would be happy if you could add tslint + hound

Comment (Thatkookooguy) [reply src/index.ts]: sure go for it.

yeah I guessed the quotes were because of a copy paste :-)

**Pull request merged**